apps/consumer-client/src/pages/pcdpass-examples/zkeddsa-proof.tsx

```diff
@@ -7,12 +7,12 @@ import {
 } from "@pcd/passport-interface";
 import { ArgumentTypeName } from "@pcd/pcd-types";
 import { SemaphoreIdentityPCDPackage } from "@pcd/semaphore-identity-pcd";
+import { generateSnarkMessageHash } from "@pcd/util";
```

Comment:
@robknight also added this function to `@pcd/util` in #556. depending on who gets their PR in first there could or could not be a merge conflict here.